packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1408,6 +1414,11 @@ class ThemeData with Diagnosticable {
   /// This is the value returned from [TooltipTheme.of].
   final TooltipThemeData tooltipTheme;
 
+  /// A theme for customizing the visual properties of [ExpansionTile]s.
+  ///
+  /// This is the value returned from [ExpansionTileTheme.of].
```

Comment:
Best to leave this sentence out, since it's not really correct. We really need to regularize the API doc for all of the FooThemeData valued properties (not necessary to do that here though).

Reply:
Thanks!